tests/test_models.py

```diff
@@ -590,6 +592,21 @@ def _test_score(estimator, p_mat, graph):
     assert np.sum(lik) == estimator.score(graph)
 
 
+class TestEdgeSwaps(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.A = er_np(100, 0.5)
+        cls.B = csr_matrix(cls.A)
+
+    def test_numpy_edge_swap(self):
+        Swapper = EdgeSwap(self.A)
+        Swapper.swap_edges()
+
+    def test_scipy_edge_swap(self):
+        Swapper = EdgeSwap(self.B)
+        Swapper.swap_edges()
```

Comment:
this isnt much testing. can we verify that degree sequence stays the same, at least?